exec/csnex/Extensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿/*****************************************************************************
+ *   File: Extensions.cs
+ * Author: Larry Frieson
+ *   Date: 2020-12-24 15:30:08
+ *   Desc: This file contains only a single class, used to extend the
+ *         functionalry of .NET core library objects.
+ ****************************************************************************/
+internal static partial class CodeExtensions
+{
+    public static byte[] CopyFrom(this byte[] source, uint start, int len)
```

Comment:
Why is this function necessary when the Array.Copy method (https://docs.microsoft.com/en-us/dotnet/api/system.array.copy?view=net-5.0) can already do this?

Reply:
Removed Extension.cs from project and use Array.Copy() static function now.